unison-cli/src/Unison/LSP/Diagnostics.hs

```diff
@@ -33,7 +33,7 @@ reportDiagnostics ::
   f Diagnostic ->
   Lsp ()
 reportDiagnostics docUri fileVersion diags = do
-  let jsonRPC = "" -- TODO: what's this for?
+  let jsonRPC = "2.0"
```

Comment:
I figured out what this is for, we just literally need to set it to `"2.0"` as per the jsonRPC spec, most LSP clients don't care, but some do.

Reply:
lol